locales/en-US/server.ftl

```diff
@@ -0,0 +1,205 @@
+shotPageShareEmail
+  .title = Share link via email
+shotPageShareLink = Get a shareable link to this shot:
+shotPagePrivacyMessage = Anyone with the link can view this shot.
+shotPageCopyImageText
+  .label = Copy image text
+shotPageConfirmDeletion = Are you sure you want to delete this shot permanently?
+// Note: {$timediff} is a placeholder for a future relative time clause like 'in 3 days' or 'tomorrow'
+shotPageExpirationMessage = If you do nothing, this shot will be permanently deleted {$timediff}.
+// Note: {$date} is a placeholder for a localized future date as returned by Date.toLocaleString.
+// For example, in en-US, {$date} could be "7/12/2017, 1:52:50 PM".
+shotPageRestoreButton = restore until {$date}
+shotPageExpiredMessage = This shot has expired.
+// Note: This phrase is followed by an empty line, then the URL of the source page
+shotPageExpiredMessageDetails = Here is the page it was originally created from:
+shotPageDeleteButton
+  .title = Delete this shot
+shotPageAbuseButton
+  .title = Report this shot for abuse, spam, or other problems
+shotPageDownloadShot
+  .title = Download
+shotPageDownload = Download
+// Note: do not translate 'Firefox Screenshots' when translating this string
+shotPageScreenshotsDescription  = Screenshots made simple. Take, save, and share  screenshots without leaving Firefox.
+shotPageUpsellFirefox = Get Firefox now
+shotPageDoesNotExpire = Does not expire
+shotPageDMCAMessage = This shot is no longer available due to a third party intellectual property claim.
+// Note: {$dmca} is a placeholder for a link to send email (a 'mailto' link)
+shotPageDMCAContact = Please email {$dmca} to request further information.
+// Note: do not translate 'Firefox Screenshots' when translating this string
+shotPageDMCAWarning = If your Shots are subject to multiple claims, we may revoke your access to Firefox Screenshots.
+// Note: {$url} is a placeholder for a shot page URL
+shotPageDMCAIncludeLink = Please include the URL of this shot in your email: {$url}
+shotPageKeepFor = How long should this shot be retained?
+// Note: shotPageSelectTime is a placeholder label for the time selection dropdown.
+shotPageSelectTime = Select time
+shotPageKeepIndefinitely = Indefinitely
+shotPageKeepTenMinutes = 10 Minutes
+shotPageKeepOneHour = 1 Hour
+shotPageKeepOneDay = 1 Day
+shotPageKeepOneWeek = 1 Week
+shotPageKeepTwoWeeks = 2 Weeks
+shotPageKeepOneMonth = 1 Month
+shotPageSaveExpiration = save
+shotPageCancelExpiration = cancel
+shotPageDoesNotExpire = does not expire
+// Note: {$timediff} is a placeholder for a future relative time clause, like "in 1 week" or "tomorrow"
+shotPageExpiresIn = expires {$timediff}
+// Note: {$timediff} is a placeholder for a past relative time clause, like "1 week ago" or "yesterday"
+shotPageExpired = expired {$timediff}
+timeDiffJustNow = just now
+// TODO: should we just have one string with plural rules?
+timeDiffOneMinuteAgo = 1 minute ago
+timeDiffMinutesAgo = {$number} minutes ago
```

Comment:
It really sucks, because I think several languages won't be able to translate this in a meaningful way. It's not a stand-alone sentence that you can turn around, e.g. "5 minutes remaining" to "Minutes remaining: 5".

I'm tempted to ask you to add proper plural forms to the FTL file, so we can start translating, and then fix the code later as soon @stasm is around. And if that's not supported by fluent-react (I'd be surprised), we can find a different solution.